profiles/base.yaml

```diff
@@ -884,8 +884,60 @@ bankbot_enabled: false
 
 # Sell loot settings
 sell_loot: true
+# To sell gems from untied gem pouches.
 sell_loot_pouch: true
+# To sell worn bundle.
 sell_loot_bundle: true
+# To sell nuggets and bars (e.g. zinc nugget and bronze bar).
+# If true, you may also want to add 'nugget' and 'bar' to `loot_additions:`
+sell_loot_metals_and_stones: false
+# The container that has the nuggets and bars (e.g. zinc nugget and bronze bar) to sell.
+sell_loot_metals_and_stones_container:
+# Specify the metals (brass, bronze, etc) and stones (jade, onyx, etc) NOT TO SELL.
+# Rare metals and stones are excluded by default.
+# Full list of metals and stones are in scripts/data/base-items.yaml
+sell_loot_ignored_metals_and_stones:
```

Comment:
This needs to be in `base-empty.yaml`

Reply:
Thanks, added to `base-empty.yaml` per https://github.com/rpherbig/dr-scripts/pull/4913/commits/860d635ec11f4e174a0c488804d2802aec443240